### PR TITLE
Remove internal TextColorChange event

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Remove internal TextColorChange event

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by lnewson — APPROVED** [review]

File: advanced/events.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ tinymce.init({
 });
 ```
 
-The following example uses a function to create a console log entry when text color is changed in the editor. This is also an example of handling an event that returns data.
+The following example uses a function to create a console log entry a command is executed in the editor. This is also an example of handling an event that returns data.
```
Comment (ltrouton) [inline advanced/events.md:29]: Should this be `console log entry when a command...`?

**Review by ltrouton — APPROVED** [review]

File: advanced/events.md [hunk 2]
```diff
@@ -142,7 +142,6 @@ The following events are provided by the {{ site.productname }} editor.
...
 | LanguageLoadError      | `{ message: string }`                                                                                                   | Fired when the editor language pack fails to load. {{site.requires_5_2v}}                                                                                                  |
 | BeforeRenderUI         | N/A                                                                                                                     | Fired before the theme UI is rendered.                                                                                                                                     |
-| TextColorChange        | `{ name: string, color: string }`                                                                                       | Fired when the text color is changed.                                                                                                                                      |
```
Comment (ashfordneil) [inline advanced/events.md:145]: What are semver's thoughts on this?

**Review by ashfordneil — APPROVED** [review]

File: advanced/events.md (hunk 2 above)
Comment (lnewson) [reply advanced/events.md:145]: Docs don't follow semver and it's not actually removed in the source, as it's an internal-only event and as such should never have been documented. Also worth noting the description for this was wrong anyways.

File: advanced/events.md (hunk 1 above)
Comment (tylerkelly13) [reply advanced/events.md]: :facepalm: I shouldn't make edits after 6pm. Thanks!

Comment (tylerkelly13) [reply advanced/events.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1658/commits/4f0273d9c520a9eaa00a46d1ff4a9a69fa8f31aa

**Review by EkimChau — APPROVED** [review]

**Pull request merged**